src/Components/SignUp.tsx

```diff
@@ -1,11 +1,46 @@
 import HumaaansWireframe from '../Images/Humaaans Wireframe.png';
 import '../Styles/SignUp.modules.css';
+import React, { useState } from 'react';
 
 interface Props {
     backToLoginHandler: () => void;
 }
 
 function SignUp({backToLoginHandler}: Props) {
+
+    const url = 'http://localhost:4000/register';
```

Comment:
try to make the domain url `http://localhost:4000`as a global variable and then import it so that we do not have to hardcode it everywhere we need to make an api call, but instead, we just import from one file.